### DISCUSSION TITLE
Other users not being added

### DISCUSSION THREAD

**Discussion by SrPeterr** [JakyeRU/Larascord#21] [Q&A]
Hi!
I'm having a bit of a problem here... I can log-in with no problems and I'm added to the db, but when I try it with a friend, nothing appends and is not added to the db, Is there i way I can see logs? I did check `storage/logs/laravel.log` and there is nothing there

**Comment by SrPeterr** [comment]
I tried reinstalling and now It doesn't even create me a DiscordController :(
And I get this message
![image](https://user-images.githubusercontent.com/21348463/149177881-feda294f-a574-4bfa-946a-b137f5ce57c4.png)

**Comment by ValentinNeaga** [comment]
Hello, @SrPeterr,

Yesterday Larascord was updated from v2.1.1 to v3.0.0 (major release). Version 3.x is not compatible with 2.x versions.
In version 3.x the `DiscordController` is not created anymore. The routes are loaded from the service provider.
You can install the latest Larascord version in a new project (which is what I recommend) or reinstall the old one using `composer require jakyeru/larascord@2.1.1`.

Also, I would recommend you to use [issues](https://github.com/JakyeRU/Larascord/issues/new/choose), not discussions. Thank you!

**Reply by SrPeterr** [reply]
> 

Fixed with Larascord v3.0.1.
Thank you!

**Comment by TFWIsaac** [comment]
Is PHP8.X required for the version 3?

**Comment by SrPeterr** [comment]
@JakyeRU if there is a new update how can I properly update Larascord? Just with `composer update`? 
Or do I need to `php artisan larascord:install` and reinstall everything again? I don't want to lose my work :(

**Reply by ValentinNeaga** [reply]
If you are using Larascord <2.x you'll have to reinstall everything. If you are using Larascord >3.x all you have to do is do `composer update` and re-publish your configuration (`php artisan larascord:publish --force`) if it has been updated. (This will overwrite your old configuration though. You should save it before running that command.)
